### DISCUSSION TITLE
Missing GitHub release for 3.15.0?

### DISCUSSION THREAD

**Discussion by wetneb** [encode/django-rest-framework#9293] [Potential Issue]
As dependabot offered me to upgrade DRF from 3.14.0 to 3.15.0, I tried looking up the release notes by going to the GitHub releases for the project, but it seems that there isn't a GitHub release for this version. Are there any plans to make one, or should I look for the release notes elsewhere?

Thank you for maintaining the DRF :)

**Comment by cclauss** [comment]
> should I look for the release notes elsewhere?

https://github.com/encode/django-rest-framework/blob/master/docs/community/release-notes.md

**Reply by TGoddessana** [reply]
I think @wetneb is talking about like this(github tag)

https://github.com/encode/django-rest-framework/releases/tag/3.14.0
